src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -965,12 +967,12 @@ void Graph::PushInputData(const std::string& name, const ov::SoPtr<ITensor>& inp
             }
         }
     } else {
-        OPENVINO_THROW("Input blob for infer '", name, "' doesn't correspond to input in network");
+        OPENVINO_THROW("Input blob for infer '", index, "' doesn't correspond to input in network");
```

Comment:
It will be better to add more info, such as node name?

Reply:
@riverlijunjie  No node info in this situation I think.